src-clean/services/api/index.js

```diff
@@ -1,17 +1,51 @@
-import axios from 'axios'
+import fetch from 'isomorphic-fetch'
+import { apiUrl } from 'config'
 
-const facade = {}
+const api = {}
+const headers = {}
 
-const api = axios.create({ baseURL: 'https://jsonplaceholder.typicode.com' })
+api.request = (endpoint, method, settings, body) => {
+  const url = (endpoint.indexOf(apiUrl) === -1) ? apiUrl + endpoint : endpoint
 
-facade.request = (config) => api.request(config)
+  return fetch(url, api.init(method, settings, body))
+    .then(api.checkStatus)
+    .then(response => response)
+    .catch(error => Promise.reject(error))
+}
+
+api.init = (method = 'GET', settings = {}, body = null) => {
```

Comment:
Could we make this method name more intuitive? At first, I thought it did initialization of the `api` object, but it seems more like `configureRequest`.